devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy

```diff
@@ -398,46 +399,74 @@ def ping() {
 	log.debug "${device.displayName} is configuring its settings"
+    
 	def request = []
-
-	//1. set association groups for hub - 2 groups are used to set battery refresh interval different than sensor report interval
-	request << zwave.associationV1.associationSet(groupingIdentifier: 1, nodeId: zwaveHubNodeId)
-	request << zwave.associationV1.associationSet(groupingIdentifier: 2, nodeId: zwaveHubNodeId)
-
-	//2. automatic report flags
-	// param 101 -103 [4 bytes] 128: light sensor, 64 humidity, 32 temperature sensor, 15 ultraviolet sensor, 1 battery sensor
-	// set value  241 (default for 101) to get all reports. Set value 0 for no reports (default for 102-103)
-	//association group 1
-	request << zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 240)
-
-	//association group 2
-	request << zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 1)
-
-	//3. no-motion report x seconds after motion stops (default 20 secs)
-	request << zwave.configurationV1.configurationSet(parameterNumber: 3, size: 2, scaledConfigurationValue: timeOptionValueMap[motionDelayTime] ?: 20)
-
-	//4. motionSensitivity 3 levels: 3-normal, 5-maximum (default), 1-minimum, 0 - disabled
-	request << zwave.configurationV1.configurationSet(parameterNumber: 4, size: 1,
-		configurationValue:
-			motionSensitivity == "normal" ? [3] :
-				motionSensitivity == "minimum" ? [1] :
-					motionSensitivity == "disabled" ? [0] : [5])
-
-	//5. Parameters 111-113: report interval for association group 1-3
-	//association group 1 - set in preferences, default 8 mins
-	request << zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: timeOptionValueMap[reportInterval] ?: (8 * 60))
-
-	//association group 2 - report battery every 6 hours
-	request << zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 6 * 60 * 60)
-
+	
+	//0. added as MSR wasn't getting detected upon pair. 
+	request << zwave.manufacturerSpecificV2.manufacturerSpecificGet()
+        //1. set association groups for hub - 2 groups are used to set battery refresh interval different than sensor report interval
+        request << zwave.associationV1.associationSet(groupingIdentifier: 1, nodeId: zwaveHubNodeId)
+
+        //2. automatic report flags
+        // param 101 -103 [4 bytes] 128: light sensor, 64 humidity, 32 temperature sensor, 15 ultraviolet sensor, 1 battery sensor
+        // set value  241 (default for 101) to get all reports. Set value 0 for no reports (default for 102-103)
+        //association group 1
+        request << zwave.configurationV1.configurationSet(parameterNumber: 101, size: 1, scaledConfigurationValue: 240)
+
+        //association group 2
+        request << zwave.configurationV1.configurationSet(parameterNumber: 102, size: 1, scaledConfigurationValue: 1)
```

Comment:
nit: looks like you've accidentally done spaces instead of tabs here

Reply:
I did a quick change to the tabs, let me know if there is anything else that i need to fix.